Tests/SwiftLintFrameworkTests/ConfigurationTests.swift

```diff
@@ -266,6 +266,7 @@ extension XCTestCase {
 }
 
 extension ConfigurationTests {
+    // swiftlint:disable:next void_return
     static var allTests: [(String, (ConfigurationTests) -> () throws -> Void)] {
```

Comment:
I don't think it'd be too hard to address this case, it'd be more a question of choosing the expected style for the type in the middle of a curried declaration. I think the current declaration would be ok, what do you think?